src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -122,7 +122,6 @@
             "MaxNumberOfRetries": 18,
             "MaxWaitTimeInSeconds": 90
         },
-        "EnableChainedSearch": true,
```

Comment:
>EnableChainedSearch [](http://example.com/codeflow?start=9&length=19)

I need it for member-match service.
Otherwise tests in paas (related to member-match) start failing